packages/ecc-utils-design/package.json

```diff
@@ -32,8 +35,9 @@
     "build": "node ../../scripts/build.js -p ecc-utils-design-",
     "dev": "concurrently -r \"npm run build -- --watch\" \"wds\"",
     "clean": "rm -rf dist node_modules custom-elements-manifest.config.js",
-    "test": "wtr --coverage",
-    "test:watch": "wtr --watch",
+    "test": "web-test-runner --group default --config ../../web-test.runner-config.mjs",
+    "test:component": "web-test-runner --watch --config ../../web-test.runner-config.mjs --group",
```

Comment:
Can see if this script is used anywhere. Is this of use?

Reply:
it allows us to run tests for individual components for example I can do `npm run test:component form` to run only tests for the form component.